Makefile

```diff
@@ -95,8 +95,9 @@ release-pachctl:
 docker-build:
 	DOCKER_BUILDKIT=1 goreleaser release -p 1 --snapshot $(GORELDEBUG) --skip-publish --rm-dist -f goreleaser/docker.yml
 
-docker-build-pipeline-build:
-	DOCKER_BUILDKIT=1 goreleaser release -p 1 --snapshot $(GORELDEBUG) --skip-publish --rm-dist -f goreleaser/docker-build-pipelines.yml
+docker-build-pipeline-build: install
```

Comment:
This looks cleaner than what we have today 👏 

Even though I don't like that we are calling `install` when version should be available. It just adds to build times. Since you are changing this, you may want to fix the places where we call these targets and remove the `make install` part before these targets.

Reply:
We should do this after you come back from the holidays. But something to think about. The docker-* targets are all used for internal testing and CI. We can put in any version number in the images ... like 9.99.9-<commitID>. This way we get rid of the whole make install step everywhere.